src/app/login-page/login-page.component.ts

```diff
@@ -1,24 +1,35 @@
 import { Component, OnInit } from '@angular/core';
 import { AuthService } from '../core/services/auth.service';
+import { UserInfo } from '../core/models';
+import { switchMap } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-login-page',
   templateUrl: './login-page.component.html',
   styleUrls: ['./login-page.component.less']
 })
 export class LoginPageComponent implements OnInit {
+  public isAuth: boolean = false;
   public username: string;
   public password: string;
   public isAuthentificated: boolean;
 
-  constructor(private authService: AuthService) { }
+  constructor(
+    private authService: AuthService,
+    private router: Router
+  ) { }
 
   public ngOnInit(): void {
     this.isAuthentificated = this.authService.isAuthentificated();
```

Comment:
Why is this here?
if we are on the login page, shouldn't we be logged out by default?

Reply:
yes, sure. You're right, this piece of code is not important. I deleted it